src/cpp/session/modules/SessionGit.cpp

```diff
@@ -296,6 +313,10 @@ Error gitExec(const ShellArgs& args,
    options.detachProcess = true;
 #endif
 
+   std::string log = core::system::getenv("RSTUDIO_GIT_LOG");
+   if (!log.empty())
```

Comment:
Nit: I think values like `0` should also not log (feel free to change that on master if you agree)

Reply:
Agreed -- made that change here (with a helper function just so we can use the same notion for 'truthiness' wherever we make that decision): https://github.com/rstudio/rstudio/commit/ab8e819b64818f060a7a9fda779a64964277a3f3